src/pages/Chat/components/MessageForm/template.ts

```diff
@@ -0,0 +1,7 @@
+export const template = `form.message-form
+     .message-form__settings
+        img(src=iconSettingFile)
+     input(placeholder=placeholder name=inputName class="message-form__input") 
+     button(type="submit" class="message-form__btn")
+      img(src=iconSubmit)
+                `;
```

Comment:
Непонятно что тут с отступами, стоит привести шаблоны в порядок!
```suggestion
export const template = `
form.message-form.message-form__settings
    img(src=iconSettingFile)
    input(placeholder=placeholder name=inputName class="message-form__input") 
    button(type="submit" class="message-form__btn")
    img(src=iconSubmit)
`;
```